app/components/institution-carousel/template.hbs

```diff
@@ -2,9 +2,9 @@
 <div id='__carousel' class="carousel slide" data-ride="carousel" data-interval="false">
     {{#if slides}}
         <div class="carousel-inner" role="listbox">
-            {{#each slides as |institutions index|}}
```

Comment:
Huh, is the problem that `institutions` already exists as a property on the component? I'm not sure I understand why this was broken, but a fix is a fix.